packages/remark-lint-maximum-line-length/index.js

```diff
@@ -111,7 +111,11 @@ function maximumLineLength(tree, file, option) {
   var lineLength
 
   // Note: JSX is from MDX: <https://github.com/mdx-js/specification>.
-  visit(tree, ['heading', 'table', 'code', 'definition', 'html', 'jsx'], ignore)
+  visit(
+    tree,
+    ['heading', 'table', 'code', 'definition', 'html', 'jsx', 'yaml'],
```

Comment:
:thinking: if yaml frontmatter is being added, should `toml` frontmatter be added as well?

Reply:
I'm happy to do so if that is preferred. I personally only use yaml for frontmatter, but can certainly understand wanting to handle both the same way. Just let me know if that's what you'd like and I can get it added too.

Reply:
I’d say add toml as well, get them both in together! 👍

Reply:
updated